docs/application/native/guides/app-management/app-controls.md

```diff
@@ -664,6 +668,120 @@ create_base_gui(appdata_s *ad)
    }
    ```
 
+<a name="action"></a>
+## Handle application control actions
```

Comment:
@hjhun Kindly maintain title case for all the headings and sub-headings, for example: Check line no. 600 that is written as "## Controlling the Launch Mode". Please re-write this heading as "## Handle Application Control Actions". For information regarding title case, please check: https://www.grammar-monster.com/lessons/capital_letters_title_case.htm

Reply:
Done

Reply:
Done